### PR TITLE
publish to maven central

### LINKED ISSUE #35: Maven central
How about publishing the library to maven central? With all dependencies?

2 more things that go in that direction:
 - lwjlg as just build dependency?
 - debloat of the build.gradle with "spooky" custom repositories and really a custom format. I mean I mean compare https://gitlab.bixilon.de/bixilon/kotlin-glm/-/blob/master/build.gradle.kts with this one.

Might even be the magic solution to #19

### PR DESCRIPTION
**Pull request by Bixilon**
Closes #35 

The published version `0.9.9.1-11` actually reflects the git tag `v0.9.9.1-build-11` and not master.

### CODE REVIEW

**Comment by elect86** [comment]
This is ok, but I'd like to have nonetheless the CI building on Win and MacOSx on commits

Don't you agree?

**Comment by Bixilon** [comment]
So, I understand that, I am doing that in other projects too. The thing with the JVM is, that it works platform independent. Other projects of mine do native function calling (I mean kool does too, but ensuring that it works is the duty of lwjgl and not glm). The build process is 100% identical on all platforms, only other natives might be linked. Testing should also be the same on all platforms.

tl;dr

I don't see the benefit, it only (despite being free for us) costs more cpu power.



(And it really has no custom task scheudling mechanisms that work different on all platforms)

**Comment by elect86** [comment]
Thanks sir
